core/src/main/ruby/jruby/ruby_implementations/Integer/times.rb

```diff
@@ -0,0 +1,9 @@
+class Integer
```

Comment:
maybe at some point the *ruby_implementations* name should make it clear that its only used when inlining?

Reply:
yeah we can change this name at any point.   I am not sure if this will eventually only be for inlining though.  Another thought about the ruby implementations is they can be specialized implementations if we know exactly how they will be used.  In that case, this could be a much more general location?